### PR TITLE
update Keen IO docs for upcoming app/partnership

### PR DESCRIPTION
**Pull request by elof**
**Description of the change**: I updated the two Keen IO docs 
**Reason for the change**: this now includes information regarding the app that we will be launching with Sendgrid next week, 9/13
**Link to original source**: https://sendgrid.com/docs/Integrate/Tutorials/analytics_with_keen_io.html

@ksigler7

### CODE REVIEW

**Comment by ksigler7** [comment]
Hey @elof - Thanks so much for doing this! We'll hold off until next week to merge this in. Can you let me know on here if I *shouldn't* publish it?

**Comment by elof** [comment]
Hi @ksigler7! This can be merged on 9/13 :)

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 1]
```diff
@@ -15,145 +15,196 @@ It only takes a few minutes to start streaming data over and once you do you can
...
-3. Enter the Keen HTTP POST URL, shown below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
-    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
-    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
-    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
-4. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check **Select All** under the *Select Actions*, or select just the ones you'd like to send to Keen IO.
 
-That’s it! Now, as your emails make their way through SendGrid, all of the event information will be posted to Keen IO. A new Event Collection, aptly named "SendGrid Email Events", will be created within your project.
+**Step 3: Set the HTTP POST URL (example below) as the endpoint where SendGrid will stream all of your email data. **
```
Comment (ksigler7) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: @elof I'm a little confused by this step - where do I set this URL? In the SG event webhook? In the Keen app?

Comment (elof) [reply source/Integrate/Tutorials/analytics_with_keen_io.md]: You get the URL in the Keen settings UI and you set the URL in the SG settings UI. Does that make sense?

Comment (ksigler7) [reply source/Integrate/Tutorials/analytics_with_keen_io.md]: It does! Do you have links handy for both of those pages? I don't currently have a Keen integration up, so I'm not sure what to look for.

Comment (elof) [reply source/Integrate/Tutorials/analytics_with_keen_io.md]: There isn't an easy way to link to where to get this without a Keen account. The link is embedded in the installation section of a project and contains a write key for auth

Comment (maggiejan) [reply source/Integrate/Tutorials/analytics_with_keen_io.md]: We just made some rad changes for readability and accuracy, @ksigler7 will this still be good to launch on Wed 9/13? Thanks!

**Comment by thinkingserious** [comment]
Hello @elof,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**